deploy/device-operator.yaml

```diff
@@ -32,14 +32,31 @@ metadata:
 # to generate the CRD definition
 
 ---
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.0
+    controller-gen.kubebuilder.io/version: v0.2.8
   creationTimestamp: null
   name: devicevolumes.local.openebs.io
 spec:
+  additionalPrinterColumns:
```

Comment:
I guess many changes are present in this file due to incorrect controller-gen version being used.